lib/fluent/plugin/exception_detector.rb

```diff
@@ -59,7 +59,7 @@ def self.supported
     ].freeze
 
     PYTHON_RULES = [
-      rule(:start_state, /Traceback \(most recent call last\)/, :python),
+      rule(:start_state, /^Traceback \(most recent call last\):$/, :python),
```

Comment:
After talking with Igor, we believe it's a broken agreement that we should be sniffing a user's custom logs for exception output, as those exceptions would have already been dealt with / logged by the user for their own understanding. I think the only true goal for this plugin is to capture exceptions that slipped past the customer's radar.